src/Altinn.Platform/Altinn.Platform.Authentication/Authentication/Model/IntrospectionRequest.cs

```diff
@@ -0,0 +1,18 @@
+namespace Altinn.Platform.Authentication.Model
+{
+    /// <summary>
+    /// Introspective request model
+    /// </summary>
+    public class IntrospectionRequest
+    {
+        /// <summary>
+        /// Gets or sets the token
+        /// </summary>
+        public string Token { get; set; }
+
+        /// <summary>
+        /// Gets or sets the token type hint
+        /// </summary>
+        public string TokenTypeHint { get; set; }
```

Comment:
We must probably be able to read the property as "token_type_hint". We can do that by defining serialization name.

Reply:
Nice catch.  My test data was incorrect. Fixed.